### ISSUE TITLE
[Question] The Wiki? How to Contribute?

### ISSUE THREAD

**Issue by Raavea** [paulevsGitch/BetterNether#407]
Hi there! 
  I hate the nether. I hate it so, so much, even (maybe especially) since the nether update.
 
  But your mod not only makes me WANT to go there, but keeps on showing me breathtaking new biomes. I love it. I only go into the nether usually to get quartz and get out, but with Better Nether I go in to explore and gather beautiful new plants. Thank you for making the Nether actually a place I want to go. I mean, I still hate the nether bc ghasts, but I spend hours exploring and burning through shears now. :')
  I can't even choose a favourite biome. I thought the jungle was my favourite (so lush) until I found a bone reef. I spent ages in that biome just looking around. Then I found the upside down forest, and wow. But then I found a mushroom forest, and those mushroom firs... <3

 The thing is, I notice your wiki is uh... kinda barren. It says on the mod FAQ (on curseforge) that we can help with the Wiki, but I don't see how. Maybe I'm dumb. Guidance?

  tldr; I love your mod, so much beautiful stuff, and I would like to be able to contribute to the wiki, only I can't figure out how.

**Comment by paulevsGitch** [comment]
Hi, it is not possible to contribute into wiki with pull requests (like with main repo), but you can write useful information inside this issue or on our Discord server, and then I can place it on wiki pages

**Comment by Raavea** [comment]
Would you like me to stick to the style of picture currently used in the Wiki, or is it okay to include screenshots of the plants in their natural environment?


**Nether Reed**
Nether reed is a beautiful and functional plant, which you will most commonly find in Nether Swampland and similar biomes. It grows much like overworld reeds, only instead of water, it must be planted near lava.
 Nether reed will grow taller (like reeds do) from a shorter plant naturally in the overworld given the proper environment - it does not require bonemeal to engourage this growth. 

 Nether reed is a requirement for some of the best Better Nether weaponry.

Tool: Hand
Drop: Itself
Stack: 64
Biomes:
Nether Swampland
Nether Swampland Terraces
Grows On:
Netherrack
